natsort/ns_enum.py

```diff
@@ -7,8 +7,6 @@
 
 import collections
 
-# NOTE: OrderedDict is not used below for compatibility with Python 2.6.
```

Comment:
I'm happy to make this change, but I'm not sure what that should be. Can you clarify what the code change is?

Reply:
Sorry about being cryptic.  I am specifically referring to `enum_fields` in the below code. It is written awkwardly because of the lack of `OrderedDict`. If `enum_fields` were an `OrderedDict` it wouldn't have to constantly be cast to `current_mapping`.

```python
# Construct the list of bitwise distinct enums with their fields.
enum_fields = [(name, 1 << i) for i, name in enumerate(enum_options)]
enum_fields.extend((name, 0) for name in enum_do_nothing)

for name, combo in enum_combos:
    current_mapping = dict(enum_fields)
    combined_value = current_mapping[combo[0]]
    for combo_name in combo[1:]:
        combined_value |= current_mapping[combo_name]
    enum_fields.append((name, combined_value))

current_mapping = dict(enum_fields)
enum_fields.extend((alias, current_mapping[name]) for alias, name in enum_aliases)

# Finally, extract out the enum field names and their values.
enum_field_names, enum_field_values = zip(*enum_fields)
```